src/EnergyPlus/AirflowNetwork/src/Properties.cpp

```diff
@@ -59,15 +59,15 @@ namespace AirflowNetwork {
     )
     {
         // Dry air thermal conductivity {W/m-K}
-        // Correlated over the range -20C to 70C
-        // Reference Cengel & Ghajar, Heat and Mass Transfer. 5th ed.
+        // Correlated over the range -73C to 76.9C
+        // Reference doi/10.1063/1.555744
 
-        Real64 const LowerLimit = -20;
-        Real64 const UpperLimit = 70;
+        Real64 const LowerLimit = -73;
+        Real64 const UpperLimit = 76.9;
 
-        Real64 const a = 0.02364;
-        Real64 const b = 0.0000754772569209165;
-        Real64 const c = -2.40977632412045e-8;
+        Real64 const a = 0.02420195316;
+        Real64 const b = 0.0000766872;
+        Real64 const c = -4.4e-08;
```

Comment:
The proposed correlation was obtained by fitting a linear regression to the values reported in Table 12 of [Kadoya et al.](https://aip.scitation.org/doi/10.1063/1.555744) between -~73 and +~77 deg. C. The figure below shows how the thermal conductivity of dry air differ from what EnergyPlus currently uses. Most of the differences occurs below -20 deg. C.

![image](https://user-images.githubusercontent.com/5149279/106955632-58047680-66ea-11eb-8a73-701cd6e444a4.png)